src/arguments.js

```diff
@@ -7,23 +10,39 @@
  * @return {Object} - argument template
  */
 const getArgumentObject = (object, context) => {
-    let argument = context.get.template.argument();
-    argument = {
+    let argument = {
         enum: getEnum(object.arg + object.description, context),
-        longName: getPropertyName(object.arg, context.regexp.argument.long),
-        shortName: getPropertyName(object.arg, context.regexp.argument.short),
+        longNames: getPropertyNames(object.arg, context.regexp.argument.long),
```

Comment:
I think we don't need to save this array of names to `longNames` or `shortNames`

Reply:
@svoboda-rabstvo it is using in 3 functions later. I think that it is more convenient to send one object `argument` that 3 elements: `object argument`, `array longNames`, `array shortNames`.